### PR TITLE
Removed VA policy schema

### PR DESCRIPTION
**Pull request by t-robalt**
Removed DatabaseVulnerabilityAssessmentProperties and all elements dependent upon it.

### CODE REVIEW

File: schemas/2017-03-01-preview/Microsoft.Sql.json [hunk 1]
```diff
@@ -326,36 +326,6 @@
       ],
       "description": "Microsoft.Sql/servers/databases/extendedAuditingSettings"
     },
-    "servers_databases_vulnerabilityAssessments": {
```
Comment (TianoMS) [inline schemas/2017-03-01-preview/Microsoft.Sql.json]: >servers_databases_vulnerabilityAssessments [](start = 5, length = 42)

this is a breaking change. Is the API already removed from service?

Comment (TianoMS) [inline schemas/2017-03-01-preview/Microsoft.Sql.json]: >servers_databases_vulnerabilityAssessments [](start = 5, length = 42)

this needs to be removed from 2015-01-01/deploymentTemplate and 2014-04-01-preview/deploymentTemplate

Comment (t-robalt) [reply schemas/2017-03-01-preview/Microsoft.Sql.json]: No, and it won't.
This is a temporary fix for a problem we are working on.

**Comment by Tiano2017** [comment]
@jaredmoo  I assume you guys are going to track the issue and add the changes back later.

**Pull request merged**